projects/Inventaire/module.nix

```diff
@@ -0,0 +1,91 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+let
+  cfg = config.services.inventaire;
+in
+{
+  options.services.inventaire = {
+    enable = lib.options.mkEnableOption "Inventaire server";
+
+    configOverridesFile = lib.options.mkOption {
+      type = lib.types.nullOr lib.types.path;
+      description = ''
+        Path to a .cjs file that exists at runtime with your desired settings overrides.
+
+        The passed path will be set up as `config/local.cjs`.
+      '';
```

Comment:
Is there some documentation that we can point users to which would tell them what the structure of this file is and what those overrides are?